Source/WebKit/Platform/ios/VideoPresentationInterfaceLMK.h

```diff
@@ -96,6 +101,9 @@ class VideoPresentationInterfaceLMK final : public WebCore::VideoPresentationInt
 #endif
 
     CompletionHandler<void(bool)> m_exitExternalPlaybackHandler;
+
+    MRUIDarknessPreference m_savedDarknessPreference;
+    BOOL m_savedAutoDimmingPreference;
```

Comment:
You should initialize these here instead of in the constructor.


```suggestion
    MRUIDarknessPreference m_savedDarknessPreference { MRUIDarknessPreferenceUnspecified };
    BOOL m_savedAutoDimmingPreference { YES };
```

Reply:
fixed this